src/js/disability-benefits/containers/FilesPage.jsx

```diff
@@ -119,12 +119,12 @@ class FilesPage extends React.Component {
                     ?
                     <div>
                       <h6 className="reviewed-file"><i className="fa fa-check-circle"></i>Reviewed by VA</h6>
-                      <p className="submission-date reviewed-file">{moment(getSubmittedItemDate(item)).format('MMM D, YYYY')}</p>
+                      <p className="submission-date reviewed-file">{moment(getItemDate(item)).format('MMM D, YYYY')}</p>
                     </div>
                     :
                     <div>
                       <h6>Submitted</h6>
-                      <p className="submission-date">{moment(getSubmittedItemDate(item)).format('MMM D, YYYY')}{item.status && ' (pending)'}</p>
+                      {!!getItemDate(item) && <p className="submission-date">{moment(getItemDate(item)).format('MMM D, YYYY')}{item.status && ' (pending)'}</p>}
```

Comment:
In this case, the date might be empty because some documents don't have a date, so we're checking that first. For reviewed items, there's always date.